app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    render plain: produce_csv(usage_statistics)
```

Comment:
Could a `respond_to` be used here to render the csv? See: https://apidock.com/rails/ActionController/MimeResponds/InstanceMethods/respond_to and https://api.rubyonrails.org/classes/ActionController/Renderers.html

Reply:
I think this is a much nicer approach.  Will update the code to have a CSV renderer and `respond_to` the relevant MIME type.